src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -912,8 +909,7 @@ ActionResult SoundEditor::potentialShortcutPadAction(int32_t x, int32_t y, bool
 		}
 
 		// For Kit Instrument Clip with Affect Entire Enabled
-		else if (setupKitGlobalFXMenu && (getCurrentOutputType() == OutputType::KIT)
-		         && (getCurrentInstrumentClip()->affectEntire)) {
+		else if (setupKitGlobalFXMenu) {
```

Comment:
setupKitGlobalFXMenu is only ever true if it's a kit and affect entire is enabled, so no need to check for kit and affect entire again...